src/app/frontend/_dark.scss

```diff
@@ -106,4 +106,9 @@ $kd-dark-theme: mat-dark-theme($kd-dark-theme-primary, $kd-dark-theme-accent, $k
   .kd-cross-line-primary {
     stroke: $accent;
   }
+
+  // this removes the border of the graphs
+  .c3-chart-arc path {
+    stroke: none;
+  }  
```

Comment:
Small nit: There's white space at the end of line 113 that needs to be removed. It's causing linting to fail.

To test locally, please run `npm run check` :) 

If you have questions please let me know. Sorry for the hurdles on a small change.

Reply:
Ah my bad. It seems to be fixed. Will run the check before creating the PR next time. Cheers!